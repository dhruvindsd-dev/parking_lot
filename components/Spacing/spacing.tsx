import { View } from "react-native";

export function S1() {
	return <View style={{ marginTop: 10 }} />;
}

export function S2() {
	return <View style={{ marginTop: 15 }} />;
}

export function S3() {
	return <View style={{ marginTop: 25 }} />;
}
