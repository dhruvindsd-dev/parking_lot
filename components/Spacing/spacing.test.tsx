import renderer from "react-test-renderer";
import { S1, S2, S3 } from "./spacing";

test("Spacing 1 Renders", () => {
	const tree = renderer.create(<S1 />).toJSON();
	expect(tree).toMatchSnapshot();
});

test("Spacing 2 Renders", () => {
	const tree = renderer.create(<S2 />).toJSON();
	expect(tree).toMatchSnapshot();
});

test("Spacing 3 Renders", () => {
	const tree = renderer.create(<S3 />).toJSON();
	expect(tree).toMatchSnapshot();
});
