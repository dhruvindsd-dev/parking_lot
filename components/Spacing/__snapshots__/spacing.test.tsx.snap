// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacing 1 Renders 1`] = `
<View
  style={
    Object {
      "marginTop": 10,
    }
  }
/>
`;

exports[`Spacing 2 Renders 1`] = `
<View
  style={
    Object {
      "marginTop": 15,
    }
  }
/>
`;

exports[`Spacing 3 Renders 1`] = `
<View
  style={
    Object {
      "marginTop": 25,
    }
  }
/>
`;
